src/schema/resolvers.ts

```diff
@@ -38,12 +39,28 @@ async function validateInput(userData: UserInput) {
   }
 }
 
+export function createToken(userID: number, rememberMe = false) {
+  return jwt.sign({ userID: userID }, 'Understanding how to fly into w@ll5', rememberMe ? { expiresIn: '7d' } : null);
+}
+
 export const resolvers = {
   Query: {
     hello: () => `Hello, Taqtiler!`,
   },
   Mutation: {
-    async createUser(_: unknown, args: CreateUserInput): Promise<UserOutput> {
+    async createUser(_: unknown, args: CreateUserInput, context: { token: string }): Promise<UserOutput> {
+      try {
+        const tokenPayload = jwt.verify(context.token, 'Understanding how to fly into w@ll5') as JWTPayload;
```

Comment:
O segredo incrivelmente específico está escrito em 2 lugares no seu código (por enquanto), mas pra garantir que ele seja o mesmo, não seria legal que ele viesse de uma const, então? Lembre que se for necessáio alterar esse segredo, vc teria que alterar em amis de um lugar, e esquecer algum deles resultaria num bug. Estando em apenas uma constante, isso não aconteceria.